[2024-04-13 21:50:00] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 21:50:05] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 21:50:10] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 21:50:15] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 21:50:20] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 21:50:25] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 21:50:27] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 905d54621bd8d9e7 [::1]:58902 failed checkpoint posthandshake: Connected to self
[2024-04-13 21:50:27] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn static dial conn 905d54621bd8d9e7 [::1]:20136 failed checkpoint posthandshake: Connected to self
[2024-04-13 21:50:30] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 21:50:35] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 21:50:40] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 21:50:45] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 21:50:50] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 21:50:55] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 21:50:57] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 905d54621bd8d9e7 [::1]:38700 failed checkpoint posthandshake: Connected to self
[2024-04-13 21:50:57] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn dyn dial conn 905d54621bd8d9e7 [::1]:20136 failed checkpoint posthandshake: Connected to self
